lib/email_tracking_retriever.py

```diff
@@ -12,12 +13,15 @@
 import lib.email_auth as email_auth
 from lib import util
 from lib.tracking import Tracking
+from lib.config import open_config
 
 _FuncT = TypeVar('_FuncT', bound=Callable)
 
 BASE_64_FLAG = 'Content-Transfer-Encoding: base64'
 TODAY = datetime.date.today().strftime('%Y-%m-%d')
 MAX_ATTEMPTS = 2
+CONFIG = open_config()
+AMAZON = CONFIG['amazon']
```

Comment:
This should be optional, and we need to be robust to cases where people don't have the config info yet. Possibly ask the user to manually log in if the amazon section isn't available in the config

Reply:
How do I make sure it's optional? Sorry still really new to python

Reply:
can just say something like 

```
# if we are at the login page
if 'amazon' in self.config:
  driver.find.....(...).send_keys(self.config['amazon']['username']
  # ditto for pw, handle otp / captcha, etc
else:
  input("please log in") # whatever the existing code does
```